ContributeDocumentation.md

```diff
@@ -0,0 +1,88 @@
+# Contributing Application Documentation Changes
+
+This document describes how to contribute changes to the ISIS Application Documentation, which is available online for the most recent ISIS release at: https://isis.astrogeology.usgs.gov/Application/index.html
+
+# Getting Started
+Documentation for ISIS applications is located within the ISIS source code and written in XML. The XML is processed to create the documentation that can be viewed on the ISIS website at: https://isis.astrogeology.usgs.gov/Application/index.html
+
+For many documentation contributions, very little understanding of XML is required. Changes in existing wording or additional information can be added where appropriate in the current documentation without needing to change any of the XML. 
+
+Since the ISIS documentation is located within the source code, the first step is to create a local copy of the ISIS source code to work with. To do the necessary initial setup, first complete the steps in [Getting Started with Github](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#getting-started-with-github), then move on to [Anaconda and ISIS3 Dependencies](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#getting-started-with-github). 
+ 
+# Making Your Documentation Change
+
+Documentation for each ISIS application is located in the same directory as the source code for the application itself and is named `name_of_application.xml` For example, to update the documentation for voy2isis, go to `ISIS3/isis/src/voyager/apps/voy2isis` in your clone and begin editing `voy2isis.xml`. To make your documentation change, open the xml file in the text editor of your choice, and edit the text to make your update. Most documentation changes will be made in the `<description>` section of the XML file, which creates the "Description" section on the built html documentation.
+
+# How to Preview Your Change
+
+In order to preview your documentation change, it is necessary to build the documentation and view it in a web browser. 
+The next step depends on whether you already have an existing ISIS build, or if you need to create 
+a new ISIS build. 
+
+## Building ISIS Documentation
+
+### Creating a New ISIS build: 
+
+Building ISIS requires that the anaconda environment you created in [Anaconda and ISIS3 Dependencies](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#anaconda-and-isis3-dependencies) be activated.
+
+Activate your anaconda environment with:
+
+`conda activate <environment-name>`
+
+* Navigate to the top-level directory of your ISIS3 clone (i.e `cd ISIS3`)
+
+* Create a `build` and an `install` directory at this level:
+  * `mkdir build install`
+
+* Navigate to the build directory: `cd build` 
+* Run the following command to configure your build:
```

Comment:
May want to refer to the main docs for this. If something changes, and we don't look everywhere for how to config we'll miss this one and it will be wrong. Suggest never duplicating docs